packages/modular-scripts/src/__tests__/app.esbuild.test.ts

```diff
@@ -84,6 +84,7 @@ describe('when working with an app', () => {
       ├─ logo192.png #1nez7vk
       ├─ logo512.png #1hwqvcc
       ├─ manifest.json #19gah8o
+      ├─ package.json
```

Comment:
It would probably be good to include the hash in the test, or at least add a test case for the package.json content.

Reply:
Done